api-spec/openapi/STAC.yaml

```diff
@@ -116,100 +116,60 @@ components:
     limit:
       name: limit
       in: query
-      description: |-
-        The optional limit parameter limits the number of items that are presented in the response document.
-
-        Only items are counted that are on the first level of the collection in the response document.
-        Nested objects contained within the explicitly requested items shall not be counted.
-
-        Minimum = 1. Maximum = 10000. Default = 10.
+      description:
+        $ref: '#/components/schemas/limit/description'
       required: false
       schema:
-        type: integer
-        minimum: 1
-        maximum: 10000
-        default: 10
+        $ref: '#/components/schemas/limit'
       style: form
       explode: false
-    page:
-      name: page
+    next:
+      name: next
       in: query
-      description: |
-        The optional page parameter returns the specified page of results
-        (with each page having size=limit).
-
-        * Minimum = 1
-        * Default = 1
+      description:
+        $ref: '#/components/schemas/next/description'
```

Comment:
I guess this works in most openAPI implementations, but it is not allowed by openAPI. The description objects (and there are more similar references here) don't allow $refs to be used. Only strings are allowed.

Reply:
The OpenAPI spec is not JSON Schema and works differently. $refs are not allowed by default, but must be allowed explicitly. See for example the [Components object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#componentsObject) where it explicitly states that Reference Objects (= $refs) are allowed. The descriptions are only of type string and don't mention Reference Object, therefore $refs are not allowed for descriptions.

Reply:
@m-mohr updated my PR. Could you take a look again?